src/libraries/System.Formats.Asn1/src/System/Formats/Asn1/WellKnownOids.cs

```diff
@@ -0,0 +1,423 @@
+        private static ReadOnlySpan<byte> Secp521r1 =>
+            new byte[] { 0x2B, 0x81, 0x04, 0x00, 0x23, };
+        private static ReadOnlySpan<byte> CommonName =>
+            new byte[] { 0x55, 0x04, 0x03, };
+        private static ReadOnlySpan<byte> SerialNumber =>
+            new byte[] { 0x55, 0x04, 0x05, };
+        private static ReadOnlySpan<byte> CountryOrRegionName =>
+            new byte[] { 0x55, 0x04, 0x06, };
+        private static ReadOnlySpan<byte> Locality =>
+            new byte[] { 0x55, 0x04, 0x07, };
+        private static ReadOnlySpan<byte> StateOrProvinceName =>
+            new byte[] { 0x55, 0x04, 0x08, };
+        private static ReadOnlySpan<byte> OrganizationName =>
+            new byte[] { 0x55, 0x04, 0x0A, };
+        private static ReadOnlySpan<byte> OrganizationalUnit =>
+            new byte[] { 0x55, 0x04, 0x0B, };
+        private static ReadOnlySpan<byte> OrganizationIdentifier =>
+            new byte[] { 0x55, 0x04, 0x61, };
+        private static ReadOnlySpan<byte> SubjectKeyIdentifier =>
+            new byte[] { 0x55, 0x1D, 0x0E, };
+        private static ReadOnlySpan<byte> KeyUsage =>
+            new byte[] { 0x55, 0x1D, 0x0F, };
+        private static ReadOnlySpan<byte> SubjectAlternativeName =>
+            new byte[] { 0x55, 0x1D, 0x11, };
+        private static ReadOnlySpan<byte> BasicConstraints =>
+            new byte[] { 0x55, 0x1D, 0x13, };
+        private static ReadOnlySpan<byte> CrlNumber =>
+            new byte[] { 0x55, 0x1D, 0x14, };
+        private static ReadOnlySpan<byte> AuthorityKeyIdentifier =>
+            new byte[] { 0x55, 0x1D, 0x23, };
+        private static ReadOnlySpan<byte> Aes128Cbc =>
+            new byte[] { 0x60, 0x86, 0x48, 0x01, 0x65, 0x03, 0x04, 0x01, 0x02, };
+        private static ReadOnlySpan<byte> Aes192Cbc =>
+            new byte[] { 0x60, 0x86, 0x48, 0x01, 0x65, 0x03, 0x04, 0x01, 0x16, };
+        private static ReadOnlySpan<byte> Aes256Cbc =>
+            new byte[] { 0x60, 0x86, 0x48, 0x01, 0x65, 0x03, 0x04, 0x01, 0x2A, };
+        private static ReadOnlySpan<byte> Sha256 =>
+            new byte[] { 0x60, 0x86, 0x48, 0x01, 0x65, 0x03, 0x04, 0x02, 0x01, };
+        private static ReadOnlySpan<byte> Sha384 =>
+            new byte[] { 0x60, 0x86, 0x48, 0x01, 0x65, 0x03, 0x04, 0x02, 0x02, };
+        private static ReadOnlySpan<byte> Sha512 =>
+            new byte[] { 0x60, 0x86, 0x48, 0x01, 0x65, 0x03, 0x04, 0x02, 0x03, };
+        private static ReadOnlySpan<byte> CabForumDV =>
+            new byte[] { 0x67, 0x81, 0x0C, 0x01, 0x02, 0x01, };
+        private static ReadOnlySpan<byte> CabForumOV =>
+            new byte[] { 0x67, 0x81, 0x0C, 0x01, 0x02, 0x02, };
+
+        internal static string? GetValue(ReadOnlySpan<byte> contents)
+        {
+            return contents switch
+            {
+                [ 0x2A, 0x86, 0x48, 0xCE, 0x38, 0x04, 0x01, ] => "1.2.840.10040.4.1",
```

Comment:
I wrote a program to generate this file, so one can generally trust that the name/dotted-decimal/encoded-contents are the same in all three places.  It didn't seem like we'd be changing it enough often enough to need to check it in... adding one OID at a time is easy to verify.

```C#
            (string Oid, string Name, byte[]? Encoded)[] data =
            {
                ("1.2.840.10040.4.1", "DSA", null),
                ("1.2.840.10040.4.3", "DSAWithSha1", null),
                ("1.2.840.10045.2.1", "EC", null),
...
```

(the `nulls` got updated by calling WriteObjectIdentifier and saving the contents slice)